src/screencast_keys/ops.py

```diff
@@ -1507,3 +1539,23 @@ def invoke(self, context, event):
         self.draw_handler_add(context)
         context.window_manager.modal_handler_add(self)
         return {'RUNNING_MODAL'}
+
+
+@BlClassRegistry()
+class SK_OT_WaitForInitialize(bpy.types.Operator):
+    bl_idname = "wm.sk_wait_for_initialize"
+    bl_label = "Screencast Keys Wait For Initialize"
+
+    initialization_handler = None
+
+    def execute(self, context):
+        cls = self.__class__
+        cls.initialization_handler = bpy.types.SpaceView3D.draw_handler_add(cls.intialization_callback, (None, None), 'WINDOW', 'POST_PIXEL')
+        print(cls.initialization_handler)
+        return {'FINISHED'}
+
+    @classmethod
+    def intialization_callback(cls, self, cotext):
+        if bpy.context.area is not None:
+            bpy.ops.wm.sk_screencast_keys('INVOKE_REGION_WIN', restart=True)
+            bpy.types.SpaceView3D.draw_handler_remove(cls.initialization_handler, 'WINDOW')
```

Comment:
Could you add an empty line.

Reply:
Changed, if you mean between 1560 and 1561 lines.

Reply:
Sorry for the unclear comment.

My suggestion is adding the empty line to the last of file.

Reply:
Ok, got it.